docs/src/migrations/sway.md

```diff
@@ -0,0 +1,92 @@
+# Sway Migrations Guide
+
+## August 16, 2024
+
+[Release v0.63.0](https://github.com/FuelLabs/sway/releases/tag/v0.63.0)
+
+### `#[namespace()]` attribute is no longer supported - [#6279](https://github.com/FuelLabs/sway/pull/6279)
+
+We no longer support the `#[namespace()]` attribute.  If you use it, notably with SRC14, you should migrate to using the `in` keyword if you want backwards compatibility.  If you just care about namespacing, you should use the new namespacing syntax.
```

Comment:
I think this should include an example of migrating the SRC14 standard since that's the most common pain point.